js/symb/taxa.sharedTaxonomyCRUD.js

```diff
@@ -83,14 +83,16 @@ async function handleFieldChange(
   submitText,
   originalForm
 ) {
-  document.getElementById("error-display").textContent = "";
+  document.getElementById("error-display").textContent = processTextContent("");
   const submitButton = document.getElementById(submitButtonId);
   submitButton.disabled = true;
   submitButton.textContent = translations.BUTTON_CHECKING;
   const isOk = await verifyLoadForm(form, silent, originalForm);
-  if (!isOk) {
+  const shouldDisableButton = false;
```

Comment:
I just hard coded this here for now. We can always make this configurable in symbini, but I figured that was not wanted at this point, given that portal-specific database configurations make this impossible even upon click of the submission button.